lib/RegisterController.js

```diff
@@ -71,3 +78,29 @@ export class RegisterController {
     }
   }
 }
+
+async function _verifyTurnstileToken({token}) {
+  // dummy secret key, load actual key from config
+  const SECRET_KEY = '1x0000000000000000000000000000000AA';
+  // TODO: load url from config
+  const url = 'https://challenges.cloudflare.com/turnstile/v0/siteverify';
```

Comment:
👍 to moving the URL to the config. They may release a `v1` at some point.

Reply:
+1 to move these things to the config. However, I'm confused about a secret key being used here in the client side code... which is not secret. It looks like we need this to hit a new endpoint that an appropriate bedrock module exposes -- which will in turn use a configured secret and turnstile URL in the backend.

I suspect we will need to integrate the token into the registration endpoint itself -- because otherwise someone can just skip it in the frontend by changing the JS, right? If the goal is to gate account registration on having a proper turnstile token, we'll need to update account registration to accept such a token and do the check on it prior to creating the account.

Reply:
@BigBlueHat yeah, I was confused where this should go, it needs to be on the server side, so I am currently moving it over to `bedrock-account-http`.